sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -292,6 +299,30 @@ public static boolean isConfigured() {
         return getInstance().isInstanceConfigured();
     }
 
+    /**
+     * Check whether app is running in App Center Test.
+     *
+     * @return true if running in App Center Test, false otherwise
+     * (and where no test dependencies in release.
+     */
+    public static boolean isRunningInAppCenterTestCloud() {
+        try
+        {
+            Bundle arguments = InstrumentationRegistryHelper.getArguments();
+            String runningValue = arguments.getString(RUNNING_IN_APP_CENTER);
+
+            return runningValue != null && runningValue == "1";
```

Comment:
Cannot use equals for string, it would work only based on luck with intern strings.
Yoda condition can be used to avoid null checking.

```suggestion
            return "1".equals(runningValue);
```